doc/manual/utilities.rst

```diff
@@ -3,25 +3,43 @@ Utilities
 
 .. Sort these tool by some subjective combination of their
    typical sequence and expected frequency of use.
+
+ARTIQ Firmware Service (AFWS) client
+------------------------------------
+
+This tool serves as a client for building tailored firmware and gateware from M-Lab's servers and downloading the binaries in ready-to-flash format. It is necessary to have a valid subscription to AFWS. Subscription also includes general helpdesk support and can be purchased or extended by contacting ``sales@``. One year of support is included with most hardware purchased from M-Labs. 
+
+.. argparse:: 
+   :ref: artiq.frontend.afws_client.get_argparser
+   :prog: afws_client 
+   :nodescription: 
+   :nodefault: 
+
+   passwd 
+      .. warning:: 
+         After receiving your credentials from M-Labs, it is strongly recommended to change your password as soon as possible. It is your responsibility to set and remember a secure password. In emergencies passwords can be reset by contacting helpdesk@.  
```

Comment:
I would skip "in emergencies" - firmware upgrades are never an emergency. "If forgotten," maybe.